scipy/stats/setup.py

```diff
@@ -28,6 +28,21 @@ def configuration(parent_package='',top_path=None):
         sources=['mvn.pyf','mvndst.f'],
     )
 
+    # Build cython rectangle demo
+    config.add_data_files('Rectangle.pxd')
+    config.add_extension(
+        'rect', sources=['rect.cxx'])
+
+    # Build hello world demo
+    config.add_data_files('HelloWorld.pxd')
+    config.add_extension(
+        'hello', sources=['hello.cxx'])
+
+    # Build BiasedUrn
```

Comment:
I saw examples of adding folders with lots of source files in other `setup.py` files (e.g. for `CKDTree`) but I imagine it would be a lot easier for someone who knows something about `numpy.distutils`. What should go here?